### PR TITLE
Update readmes to match common design (Network Analysis category) 1/2

### PR DESCRIPTION
**Pull request by JonLavi**

### CODE REVIEW

File: network_analysis/closest-facility-static/README.md [hunk 1]
```diff
@@ -1,36 +1,30 @@
...
 
 Find routes from several locations to the respective closest facility.
 
-![](ClosestFacilityStatic.png)
+![Image of find closest facility static](ClosestFacilityStatic.png)
 
 ## Use case
 
-Quickly and accurately determining the most efficient route between a location and a facility is a frequently encountered task (e.g. emergency services).
+Quickly and accurately determining the most efficient route between a location and a facility is a frequently encountered task. For example, a city's fire department may need to know which firestations in the vicinity offer the quickest routes to multiple fires. Solving for the closest fire station to the fire's location using an impedance of "travel time" would provide this information.
```
Comment (Rachael-E) [inline network_analysis/closest-facility-static/README.md]:
```suggestion
Quickly and accurately determining the most efficient route between a location and a facility is a frequently encountered task. For example, a city's fire department may need to know which fire stations in the vicinity offer the quickest routes to multiple fires. Solving for the closest fire station to the fire's location using an impedance of "travel time" would provide this information.
```

File: network_analysis/closest-facility-static/README.md [hunk 2]
```diff
@@ -1,36 +1,30 @@
...
-To display a `ClosestFacilityRoute` between several incidents and facilities:
-
-1. Create a `ClosestFacilityTask` using a Url from an online service.
+1. Create a `ClosestFacilityTask` using a URL from an online service.
 2. Get the default set of `ClosestFacilityParameters` from the task: `closestFacilityTask.createDefaultParametersAsync().get()`.
-3. Build a list of all Facilities and Incidents:
-    * Create a `FeatureTable` using `ServiceFeatureTable(Uri)`.
-    * Query the `FeatureTable` for all `Feature`s using `.queryFeaturesAsync(queryParameters)`.
-    * Iterate over the result and add each `Feature` to a `List`, instantiating the feature as a `Facility` or `Incident`.
+3. Build a list of all `Facilities` and `Incidents`:
```
Comment (Rachael-E) [inline network_analysis/closest-facility-static/README.md]: adjusting this to reflect the API class name even though grammatically it looks wrong!
```suggestion
3. Build a list of all `Facility`s and `Incident`s:
```

File: network_analysis/closest-facility-static/README.md [hunk 3]
```diff
@@ -1,36 +1,30 @@
...
+  * Create a `FeatureTable` using `ServiceFeatureTable(Uri)`.
+  * Query the `FeatureTable` for all `Features` using `queryFeaturesAsync(queryParameters)`.
```
Comment (Rachael-E) [inline network_analysis/closest-facility-static/README.md]:
```suggestion
  * Query the `FeatureTable` for all `Feature`s using `queryFeaturesAsync(queryParameters)`.
```

File: network_analysis/find-route/README.md [hunk 4]
```diff
@@ -1,36 +1,35 @@
...
+![Image of find route](FindRoute.png)
+
+## Use case
+
+Find routes with driving directions between any number of locations. You might use the ArcGIS platform to create a custom network for routing on a private roads.
 
 ## How to use the sample
 
-For simplicity, the sample comes loaded with a start and end stop. You can click on the Find route button to get a route between these stops. Once the route is generated, the DirectionMessages show step by step directions in the directions list.
+For simplicity, the sample comes loaded with a start and end stop. You can tap on the 'Find Route' button to display a route between these stops. Once the route is generated, turn-by-turn directions are shown in a list. You can use the 'Reset' button to reset the sample.
```
Comment (Rachael-E) [inline network_analysis/find-route/README.md]: we've been using double quotes elsewhere for buttons so might as well continue that!
```suggestion
For simplicity, the sample comes loaded with a start and end stop. You can tap on the "Find Route" button to display a route between these stops. Once the route is generated, turn-by-turn directions are shown in a list. You can use the "Reset" button to reset the sample.
```

File: network_analysis/find-route/README.md [hunk 5]
```diff
@@ -1,36 +1,35 @@
...
-2. Add the map to the `MapView`, `MapView.setMap()`.
-3. Create a route task using an URL from an online service.
-    * online route task solves a route from an online route service
-4. Set the `RouteParameters` to find a route between the `Stop`s.
-5. Get the route using th. `RouteTask.solveAsync(routeParameters)`.
-6. Display the route by adding it to a `GraphicsOverlay.getGraphics().add()`.
-7. Show the step by step `DirectionMessage`s using the `Route.getDirectionManeuvers()` method.
+1. Create a `RouteTask` using a URL to an online route service.
+2. Generate default `RouteParameters` using `routeTask.createDefaultParametersAsync()`.
+3. Set `returnDirections` on the parameters to true.
```
Comment (Rachael-E) [inline network_analysis/find-route/README.md]: updating this to reflect correctly the name of our method
```suggestion
3. Use `setReturnDirections` on the parameters and set to true.
```

File: network_analysis/find-route/README.md [hunk 6]
```diff
@@ -1,36 +1,35 @@
...
+4. Add `Stop`s to the parameters `stops` collection for each destination.
```
Comment (Rachael-E) [inline network_analysis/find-route/README.md]: does this suggestion make sense? just want to get the API ref names right here
```suggestion
4. Use `setStops` on the parameters to add `Stop`s to its stops collection for each destination.
```

**Review by Rachael-E — COMMENTED** [review]
minor things and a suggestion to look over!

File: network_analysis/closest-facility-static/README.md (hunk 2 above)
Comment (JonLavi) [reply network_analysis/closest-facility-static/README.md]: Yeah the parsed MD looks a bit weird, but it's definitely better the way you suggest.

**Review by Rachael-E — APPROVED** [review]

**Pull request merged**